modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -607,12 +597,25 @@ Either use xref::indexes-for-search-performance.adoc[] instead, or remove the of
 .Error message
 [source, error]
 ----
-Unable to create Constraint( name='friends', type='RELATIONSHIP_UNIQUENESS', schema=()-[:FRIENDS_WITH {nickname}]-() ):
-Both Relationship(0) and Relationship(1) have the type `FRIENDS_WITH` and property `nickname` = 'Mimi'
+Unable to create Constraint( name='series_title', type='RELATIONSHIP UNIQUENESS', schema=()-[:SEQUEL_OF {seriesTitle}]-() ):
+Both Relationship(0) and Relationship(1) have the type `SEQUEL_OF` and property `seriesTitle` = 'Chronicles of Ancient Darkness'
 ----
 
 ======
-////
+
+The constraint creation fails on the first offending relationships that are found.
```

Comment:
```suggestion
The constraint creation fails on the first offending relationship that is found.
```

Reply:
```suggestion
The constraint creation will fail on the first offending relationships that are found.
```
the issue is two relationships having the same `seriesTitle` so keeping the plural might be nice? (would also need updating for the node versions if we want to update the comment)

Reply:
How about "the constraint creation fails on the first offending relationship that is found"?

Reply:
The error reports that `relationship(0)` and `relationship(1)` has the same value for the property so to me it feels odd to say `offending relationship` in singular...

Reply:
Ok, I see your point - had not looked properly at the error message again. From my POV, I think your phrasing makes sense, then, and this PR is good to be merged.

Reply:
Ah, right. Agreed, this makes sense to me: 

`The constraint creation will fail on the first offending relationships that are found.`